integration_tests/dbt_project.yml

```diff
@@ -10,6 +10,9 @@ dispatch:
   - macro_namespace: dbt_expectations
     search_order: ['google_ads_source', 'dbt_expectations']
 
+models:
```

Comment:
What would the effort be to add some of the passthrough columns in this integration test for one or some of our end models. This way we can always test the compatibility on subsequent releases.

Reply:
added!